packages/shopping/src/services/email.service.ts

```diff
@@ -0,0 +1,41 @@
+import {bind, BindingScope} from '@loopback/core';
+const nodeMailer = require('nodemailer');
+import {EmailTemplate, NodeMailer, User} from '../models';
+
+@bind({scope: BindingScope.TRANSIENT})
+export class EmailService {
+  private static async setupTransporter() {
+    return nodeMailer.createTransport({
+      host: process.env.SMTP_SERVER,
+      port: process.env.SMTP_PORT,
```

Comment:
This was the problem with type signature.  Always be weary of the types that you get from env. vars. in TypeScript.

```suggestion
      port: Number(process.env.SMTP_PORT) || 0,
```

Reply:
for port I would do `+(process.env.SMTP_PORT)` to covert to number